sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -427,15 +508,11 @@ export class ManagementClient extends LinkEntity {
         this._context.namespace.connectionId,
         request.body
       );
-      log.mgmt(
-        "[%s] Acquiring lock to get the management req res link.",
-        this._context.namespace.connectionId
-      );
-      await defaultLock.acquire(this.managementLock, () => {
-        return this._init();
-      });
 
-      const result = await this._mgmtReqResLink!.sendRequest(request);
+      const result = await this._acquireLockAndSendRequest(request, timeoutInMs!, {
+        abortSignal: undefined,
+        requestName: undefined
+      });
```

Comment:
Why aren't the below 3 methods using the same format?
- renewLock
- updateDispositionStatus
- listMessageSessions

The last two can use the default timeout value we have from the constants file in core-amqp

Reply:
I did not touch renewLock and updateDispositionStatus since they come under message methods which I thought to be beyond the scope of this PR.
listMessageSessions method is not present under sessionReceiver (used to be commented under sessionReceiver I think) - so ignored that.

Now.. updated `renewLock` and `updateDispositionStatus` with default timeouts and `listMessageSessions` with a timeout parameter like the rest of the methods.

Reply:
The 3 operations are using the default retry options in Track 1 anyway. So, even though the change you made now is not making any net difference to functionality of these 3 methods, it definitely makes things consistent in this file